<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-DNPUUSR" failures="0" tests="3" name="Default test" time="0.098" errors="0" timestamp="2025-06-04T18:14:37 IST">
  <testcase classname="api.tests.userTest.UpdateUserTests" name="@BeforeClass setup" time="0.026">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;api.payload.User_Payload.getUsername()&amp;quot; because &amp;quot;oldUserPayload&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "api.payload.User_Payload.getUsername()" because "oldUserPayload" is null
at api.tests.userTest.UpdateUserTests.setup(UpdateUserTests.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="api.tests.userTest.UpdateUserTests" name="TC006" time="0.001">
    <skipped/>
  </testcase> <!-- TC006 -->
  <testcase classname="api.tests.userTest.UpdateUserTests" name="TC007" time="0.0">
    <skipped/>
  </testcase> <!-- TC007 -->
  <testcase classname="api.tests.userTest.UpdateUserTests" name="TC008" time="0.0">
    <skipped/>
  </testcase> <!-- TC008 -->
</testsuite> <!-- Default test -->
